### DISCUSSION TITLE
Exclude Dolby Vision in Search

### DISCUSSION THREAD

**Discussion by McEnnes** [pymedusa/Medusa#10702] [Q&A]
When i play a 4k episode on my TV, it happens regularly that the colours are not right, they are Pink and Green.
As i understand this has to do with the Dolby Vision format. Apparently Samsung TV`s do not support Dolby Vision.
Is it  possible to exclude Dolby Vision files in the search parameters in Medusa? The way i do it  now is that i have to find another episode manually, so the option to exclude would be great! Since there are a lot of users who own a Samsung TV, i imagine there are more users who want this to be possible, or is it already and am i missing something?

**Comment by p0psicles** [comment]
You can set ignored words. So if there are specific tags in the release title. You can add those.

**Reply by McEnnes** [reply]
Thanks for the reply! 
Sometimes there is DV and sometimes there is DOVI in the release name, so i will add those. 
I am under the impression that there are not always specific tags, but i will start with this and see how this works.
Again, thanks, i appreciate this!
